lib/Elastica/Connection.php

```diff
@@ -38,6 +38,12 @@ class Connection extends Param
     const TIMEOUT = 300;
 
     /**
+     * Number of seconds after a connection timeout occurs for every request
+     * Use a small value if you need a fast response in case of dead servers.
+     */
+    const CONNECT_TIMEOUT = 5;
```

Comment:
This means setting the timeout for all clients that don't implement it to 5 seconds. Isn't the default (without setting it) 300? So it would make sense to keep the behaviour the same as before to set it to 300? Or what is the default currently used?